test/data/manifests/fedora-ostree-oci-bootiso.mpp.yaml

```diff
@@ -348,6 +209,12 @@ pipelines:
             - qemu-net
           install:
             - /.buildstamp
+      - type: org.osbuild.kickstart
+        options:
+          path: /usr/share/anaconda/interactive-defaults.ks
```

Comment:
I think the reason we chose to use our own kickstart path instead of interactive-defaults is so that we can automate installations by adding a couple of more commands (for testing).  It also has some other differences IIRC, like letting you change things that are already defined in the ks file.

This isn't an issue here, but will be important to figure out when we add support in composer.

Reply:
First things first: I took the [lorax template for Silverblue](https://pagure.io/fedora-lorax-templates/blob/main/f/ostree-based-installer/lorax-configure-repo.tmpl) as the inspiration, so that's primarily why I switched to interactive defaults.

I vaguely remember from our discussions with the installer team is that for UI-based installation, we should be using interactive defaults. I think that the reason is the new web UI (not used here, though) doesn't work when a kickstart is passed in, because it turns anaconda into a "kickstart mode".

For fully automated installation, you can still do a very similar workflow as you suggested: You just need to copy the interactive defaults first, make the partial kickstart a fully specified one (this one will fail on undefined storage AFAIK), and pass the right kernel arg.

Reply:
I also summarized my comment in the last commit message.